components/input/Input .tsx

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+import styles from "./Input.module.css";
+
+export default function Input() {
+  const [name, setName] = useState("");
```

Comment:
Storybook is intended for static preview only. Functions and hooks are only used during assembly. I think.